packages/perseus/src/widgets/label-image/validate-label-image.ts

```diff
@@ -0,0 +1,25 @@
+import type {PerseusScore} from "../../types";
+import type {PerseusLabelImageUserInput} from "../../validation.types";
+
+function validateLabelImage(
+    userInput: PerseusLabelImageUserInput,
+): Extract<PerseusScore, {type: "invalid"}> | null {
+    let numAnswered = 0;
+    for (let i = 0; i < userInput.markers.length; i++) {
+        const userSelection = userInput.markers[i].selected;
+        if (userSelection && userSelection.length > 0) {
+            numAnswered++;
+        }
+    }
```

Comment:
I re-worked the logic a bit here so as not to need scoringData. This should be the same logic; it just doesn't use `scoreMarker`.